doc/can-query-logic.md

```diff
@@ -432,52 +506,37 @@ For example, you might want to represent a date with a string like:
 
 The following creates a `DateStringSet` that translates a date string to a number:
 
-```js
-class DateStringSet {
-    constructor(value){
-        this.value = value;
-    }
-    // used to convert to a number
-    valueOf(){
-        return new Date(this.value).getTime()
-    }
-}
-```
+@sourceref ./can-query-logic-models/date-string-example.js
+@codepen
 
 These classes must provide:
 
 - `constructor` - initialized with the the value passed to a comparator (ex: `"Wed Apr 04 2018 10:00:00 GMT-0500 (CDT)"`).
-- [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/valueOf valueOf] - return a string or number
-  used to compare (ex: `1522854000000`).
+- [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/valueOf valueOf] - return a string or number used to compare (ex: `1522854000000`).
+- `Symbol.for("can.serialize")` - returns a string or number to compare against [can-data-types] for the query.
```

Comment:
It doesn't use`Symbol.for("can.serialize")` for comparisons.  It uses this to serialize instances of the `DateStringSet` back into a plain object or value for the query.